3.10/release-notes-api-changes310.md

```diff
@@ -25,4 +29,7 @@ integrations for ArangoDB 3.10.
 
 ### Endpoints removed
 
+Since ArangoDB 3.8, there were two APIs for retrieving the metrics in two different formats: `/_admin/metrics` and `/_admin/metrics/v2`. The metrics API v1 (`/_admin/metrics`) was deprecated in 3.8 and usage of `/_admin/metrics/v2` was encouraged.  
```

Comment:
I would rather put this under "Endpoint return values changed", because the 2 endpoints still exist in 3.10.
However, `/_admin/metrics` changed its return values in 3.10.

Reply:
Done